### ISSUE TITLE
Translations for facets

### ISSUE THREAD

**Issue by Msch0150** [ewg118/numishare#111]
Is it possible to translate the subjects?
I tried the following:

Add in Admin UI > Modify settings > Factes > + : fmrdFindComplex_facet

Add in coin xml > 
```
<subject localType="fmrdFindComplex">IV 3456</subject>
```

Reindex the coins. In the public UI I do see:
```
  Subjekte
    fmrdFindComplex: "IV 3456" (as link)
```
I can click on the link and see all items with "IV 3456". So far so good.
But I want "fmrdFindComplex" to appear as "FMRD Fundkomplex". So I tried in ./numishare/ui/xslt/functions.xsl:
```
...
<xsl:when test="$label='fmrdFindComplex'">FMRD Fundkomplex</xsl:when>
...
```
but unfortunately that does not work. Any ideas?

**Comment by ewg118** [comment]
Yes, it is possible. There are actually two different translation functions in that XSLT document: one which is for NUDS elements and fields pertaining to data (xlink:roles, localTypes, etc.) and the other function is for cues in the UI. The function you're looking for is regularize_node():

Try inserting your translation both into the German conditional: https://github.com/ewg118/numishare/blob/c36a00082f63e3d400cbd36c9ac48af012c0e084/ui/xslt/functions.xsl#L168

And also in the default `xsl:otherwise`, for English:

https://github.com/ewg118/numishare/blob/master/ui/xslt/functions.xsl#L1983

**Comment by Msch0150** [comment]
Thanks for the answer. I added the translations as suggested. Unfortunately it does not work for my case. 
The translation works if I exchange 

<subject localType="fmrdFindComplex">myComplex</subject>

by

<fmrdFindComplex>myComplex</fmrdFindComplex>

But then I cannot search for it.
Anyhow, I think I found a solution. I never realized for what it is about until now:

Admin UI > Modify settings > Contents, Local Types and Symbol Positions

Local Types + > Attribute Value "fmrdFindComplex", Textual Label "FMRD find complex" (English)

+Label > Textual Label "FMRD Fundkomplex" (German)

Add in Admin UI > Modify settings > Factes > + : fmrdFindComplex_facet

Out of the box functionality. Cool. For me this is fine.
